tests/e2e/models/BUILD.bazel

```diff
@@ -51,6 +52,27 @@ iree_lit_test_suite(
     ],
 )
 
+# Larger tests requiring tags skipping them in slow configurations such as
+# sanitizers.
+iree_lit_test_suite(
+    name = "lit_large",
```

Comment:
I've been proposing a more generic "heavy compute" tag that would cover that kind of use case, as well as most instances of what currenly apply `noriscv`, `noasan`, `notsan`, etc.  The main counter-argument was that that's what the existing Bazel test rules' timeout/size arguments are for, and my own counter to that was that these are essentially unusable because they are specified to mean specific timeouts which only make sense in a setting where all test hosts have the same order of magnitude of performance, i.e. this is an google-ism. The conversation died out at that point, but happy to revive it :-)

Reply:
I think we actually do map the timeout tag to CMake: https://github.com/openxla/iree/blob/6a87b03ce6df3359cd548c920319e86439943cac/build_tools/bazel_to_cmake/bazel_to_cmake_converter.py#L66

There's also the issue that there's a difference between taking a long time to compile and taking a long time to run. They tend to be correlated, but aren't the same thing.

The test sizes actually only imply a default timeout selection and the timeouts are a symbolic mapping, so you can change what actual value they are on different platforms. Then it just becomes a thing where you assume that tests have approximately the same relative latency across hosts, which also isn't true, but is more broadly applicable and would also apply to your proposed tag. The main problem is that the attribute is non-configurable because configuration wasn't built into Blaze in the beginning (and that is very much a Google historical artifact). Google now has some internal definitions of test sizes that I think are also kind of googleisms and of course don't correspond to anything in the Bazel documentation. I've mostly given up on the traditional Bazel test "sizes" at this point: they conflate too many things and have conflicting definitions. So I think we should use timeout for this purpose. It at least means something specific.

Reply:
ah ok, thanks for the explanation (main thing new to me was " the timeouts are a symbolic mapping, so you can change what actual value they are on different platforms"). OK, then.